sound/soc/sof/topology.c

```diff
@@ -1098,6 +1098,9 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 	/* get volume control */
 	scontrol = sm->dobj.private;
 
+	/* set cmd for pga kcontrol */
+	scontrol->cmd = SOF_CTRL_CMD_VOLUME;
```

Comment:
I've lost track of volume vs. mute, does this also restore the mute status if it's set before suspend?

Reply:
@plbossart, in the current SOF driver, pga widgets can only have one control. When we add the mute switch, we will also need to handle the switch command for it.

Reply:
ok